manifests/charts/dash/values.yaml

```diff
@@ -20,6 +20,10 @@ kubesec:
     password: ""
     secretRef: ""
 
+falco:
```

Comment:
Lets add a comment explaining as well as update the advanced install section of the docs